manifests/runtimeclass.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: node.k8s.io/v1
+kind: RuntimeClass
+metadata:
+  name: nvidia
+handler: nvidia
+---
+apiVersion: node.k8s.io/v1
+kind: RuntimeClass
+metadata:
+  name: nvidia-experimental
+handler: nvidia-experimental
+---
+apiVersion: node.k8s.io/v1
+kind: RuntimeClass
+metadata:
+  name: wasmtime
+handler: wasmtime
+---
+apiVersion: node.k8s.io/v1
```

Comment:
Do you want to pull the wasmtimed runtimeclass until that's working better or leave it in?

Reply:
oh I'm not planning on merging any of this at the moment, I'm going to keep it around as-is while upstream works on the shims.

Reply:
In that case I may try to add "runu"